README.md

```diff
@@ -17,17 +17,20 @@ Vue.use(components);
 ```
 
 ## Development
-```
+This repo was initialized using npm v7 which changes the `package-lock.json` file format changed significantly. Before installing node modules, make sure you have npm v7 installed.
```

Comment:
Nitpick: the first sentence could be a little cleaner.
`This repo was initialized using npm v7 which changed the `package-lock.json` file format significantly.` ☺️